biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -1977,6 +1979,8 @@ subroutine create_patch(currentSite, new_patch, age, areap, label)
     allocate(new_patch%sabs_dif(hlm_numSWb))
     allocate(new_patch%fragmentation_scaler(currentSite%nlevsoil))
 
+    allocate(new_patch%tveg24)
+    call new_patch%tveg24%InitRMean(ema_24hr) ! No initial value
```

Comment:
can we initialize new patches to have the running-mean values of their donor patch?

Reply:
Currently it is designed to fail if the mean is requested before it has a usable value.  But as of right now, we are not using the tveg24 value before it has experienced some of its own energy physics.  We could add an initial value, but the tradeoff is that we are adding arguments to the patch creation that we might not need, adding unnecessary information.  We can always add that functionality later if someone generates a scenario where we need a mean value immediately after the entity has spawned.
Note also that the InitRmean() function does have the optional initialization value coded in.  As an example, this will be necessary if someone wants to do a fixed window 24 hour average, and the model was started mid-way through the day.

https://github.com/rgknox/fates/blob/fates_runmean_vars/main/FatesRunningMeanMod.F90#L154-L155